src/user/committee.rs

```diff
@@ -193,9 +193,15 @@ impl CommitteeSet {
         &self.provisioners
     }
 
-    fn get_or_create(&mut self, cfg: sortition::Config) -> &Committee {
+    fn get_or_create(&mut self, cfg: &sortition::Config) -> &Committee {
         self.committees
-            .or_insert_with(|| Committee::new(self.this_member_key, &mut self.provisioners, cfg))
             .entry(cfg.clone())
+            .or_insert_with_key(|config| {
+                Committee::new(
+                    self.this_member_key.clone(),
+                    &mut self.provisioners,
+                    config.clone(),
```

Comment:
The former solution is also idiomatic. There is no reason to borrow `cfg` as it is always copied.